website/src/pages/chat/[id].tsx

```diff
@@ -45,19 +44,14 @@ const Chat = ({ id, modelInfos }: ChatProps) => {
 
 Chat.getLayout = getDashboardLayout;
 
-export const getServerSideProps: GetServerSideProps<ChatProps, { id: string }> = async ({
-  locale = "en",
-  params,
-  req,
-}) => {
+export const getServerSideProps: GetServerSideProps<ChatProps, { id: string }> = async ({ locale = "en", params }) => {
   if (!isChatEnabled()) {
     return {
       notFound: true,
     };
   }
 
-  const token = await getToken({ req });
-  const client = createInferenceClient(token);
+  const client = new OasstInferenceClient();
```

Comment:
Since we do all everything on the client, we should probably move this to the client at some point

Reply:
For API the doesn't require auth, I want to do it in server side. It must more performance and the logic on the client less complicated (loading/error state).